db/people.yml

```diff
@@ -1955,21 +1955,35 @@ iadnan:
     - Ibrahim Adnan
     - Ibrahim, A
     - Ibrahim ALIbrahim
-  avatar: false
+  avatar: https://github.com/EngrIbrahimAdnan.png
   bio: Ibrahim is Masters of Electrical engineering student from kuwait. He did his undergraduate at the
     university of Portsmouth, United Kingdom. After graduating in 2021, he worked as a network
     engineer at kuwait's ministry of interior for 7 months before joining columbia university. Upon
     graduating with a master's degree, Mr. ALIbrahim plans on pursuing a PhD at columbia university
     focusing on the area of Artificial intelligence and its trustworthiness in critical applications
   education:
+    - institution: Columbia University
```

Comment:
please check the indentation here.  Also remove blank lines.  indentation implies meaning in yaml.  Please spend a few mins reading about it to help, but also follow the  other entries carefully.

Reply:
On it, Simon!